scripts/bootstrap.sh.in

```diff
@@ -29,6 +37,50 @@ declare -A GPGCHECK_REPOSITORIES=(
     [metalk8s-scality]=0
 )
 
+run() {
+    local name=$1
+    shift 1
+
+    echo -n "> ${name}..."
+    local start
+    start=$(date +%s)
+    set +e
+    "$@" > "${TMPFILES}/out" 2>&1
```

Comment:
Could we toggle this redirection based on environment variable/CLI arg? Might be useful to follow what the bootstrap is doing, especially if we are running Salt with `-l debug`, or if we improve long-running commands to output progress information.

Reply:
Makes sense, will look into that.